### PR TITLE
feat(cluster): add access button to creating cluster

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

[> Link to the JIRA ticket
](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-580)

- Add a button to create a Cluster
- Add redeploy action for "cluster updated" toast
Capture d’écran 2023-02-09 à 10 38 35

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [[PR] staging - feat(cluster): add access button to creating cluster - 2023-02-08T14:10:21Z](https://new.console.qovery.com/organization/141c07c8-0dd9-4623-983b-3fdd61867255/project/977bd069-20f7-4b24-bd95-28b44174cf43/environment/494353f7-3a89-4270-aa5a-40bfeb75d938/services/general)

 Another comment will be posted when deployments are terminated

**Comment by evoxmusic** [comment]
Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://z494353f7-zcbe69165-gtw.za8ad0659.bool.sh)
:point_right: [storybook](https://z494353f7-z7939396a-gtw.za8ad0659.bool.sh)

File: libs/pages/clusters/src/lib/ui/page-clusters-general/page-clusters-general.tsx [hunk 1]
```diff
@@ -20,12 +23,7 @@ export function PageClustersGeneral(props: PageClustersGeneralProps) {
             <h1 className="h5 text-text-700 mb-2">Manage your clusters</h1>
             <p className="text-text-500 text-xs">Manage your infrastructure across different Cloud providers.</p>
           </div>
-          <Button
-            iconRight={IconAwesomeEnum.CIRCLE_PLUS}
-            onClick={() =>
-              window.open(`https://console.qovery.com/platform/organization/${organizationId}/settings/clusters`)
-            }
-          >
+          <Button iconRight={IconAwesomeEnum.CIRCLE_PLUS} link={goToCreateCluster}>
```
Comment (bdebon) [inline libs/pages/clusters/src/lib/ui/page-clusters-general/page-clusters-general.tsx:26]: 🎉

File: libs/shared/ui/src/lib/components/buttons/button/button.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Link } from 'react-router-dom'
+import { Link as LinkRouter } from 'react-router-dom'
```
Comment (bdebon) [inline libs/shared/ui/src/lib/components/buttons/button/button.tsx:1]: Aaah you mean that before this fix, it was calling itself in loop? Or? Why did you have to rename it that way?

**Review by bdebon — APPROVED** [review]
Finally to button is wired!! 🎉

**Review by bdebon — APPROVED** [review]
Perfect, everything works well and tests are on point! Good job Remicho!

**Pull request merged**